NightscoutServiceKit/NightscoutService.swift

```diff
@@ -21,12 +21,22 @@ public final class NightscoutService: Service {
 
     public weak var serviceDelegate: ServiceDelegate?
 
-    public var siteURL: URL?
+    public var siteURL: URL? {
+        didSet {
+            updateUploader()
+        }
+    }
 
-    public var apiSecret: String?
+    public var apiSecret: String? {
+        didSet {
+            updateUploader()
+        }
+    }
     
     public var isOnboarded: Bool
 
```

Comment:
otpManager is our wrapper around OneTimePassword that validates our OTP codes from push notifications.